docs/setup.html

```diff
@@ -231,19 +178,18 @@
 
 		
 		<section class="comment">
-			<h2>Installation</h2>
-<p>Install <a href="http://nodejs.org/">Node.js</a> on your
-computer. Open a console to your project. Use <a href="https://www.npmjs.org/">npm</a> to
-install DocumentJS:</p>
+			<p>This is an in-depth guide to setting up DocumentCSS with all of the bells and whisels. For this we will be installing up <a href="http://documentjs.com/">DocumentsJS</a> which powers DocumentCSS.</p>
```

Comment:
*whistles;  And I don't think you need the word "up" just 'we will be installing DocumentJS'